### DISCUSSION TITLE
Task 1: Cauchy Matrix LAB 02.02-Numpy

### DISCUSSION THREAD

**Discussion by jaayg23** [rramosp/ai4eng.v1#121] [02 - PYTHON]
Buenas, quisiera que alguien me ayudara con la tarea 1 ya que he intentando de muchas maneras que la plataforma me califique de manera correcta hago las pruebas manuales y me salen valores esperados pero la plataforma siempre me coloca 0 por la excepción a generar 

```
def cauchy(x, y):
  r =np.divide(1,x.reshape((-1,1))-y)
  if np.any(np.isinf(r)):
    raise ZeroDivisionError
  return r
```
Si pudieran ayudarme lo agradecería un montón.

**Comment by Felipe-RA** [comment]
Saludos jaayg23

Con respecto a tu problema, te sugiero una lectura más cuidadosa del enunciado en el ejercicio. Especialmente en la siguiente nota:

>  NOTE: if there is any division by zero, you must raise a <u>**ValueError**</u> exception.

Fijate la diferencia entre la excepción que tú alzas y la que se te indica que debes alzar.

Cualquier cosa por aquí estamos,

Hasta pronto!

**Reply by jaayg23** [reply]
Felipe muchísimas gracias, era eso.
